README.md

```diff
@@ -1,5 +1,5 @@
 <p align="center">
-    <img width="200" alt="Alacritty Logo" src="extra/logo/compat/alacritty-term+scanlines.png">
+    <img width="200" alt="Alacritty Logo" src="https://github.com/alacritty/alacritty/blob/master/extra/logo/compat/alacritty-term+scanlines.png">
```

Comment:
You should probably use this URL to make everything work properly:

```suggestion
    <img width="200" alt="Alacritty Logo" src="https://raw.githubusercontent.com/alacritty/alacritty/master/extra/logo/compat/alacritty-term%2Bscanlines.png">
```

That's the direct URL tot he image.